packages/webcomponents/src/components/payments-list/payments-list.tsx

```diff
@@ -90,6 +88,7 @@ export class PaymentsList {
       <payments-list-core
         getPayments={this.getPayments}
         onError-event={this.handleErrorEvent}
+        exportparts={`${tableExportedParts},label, pagination-buttons-container`}
```

Comment:
I'm a little confused looking at this exportparts line written here as an attribute for payments-list-core, instead of being exorted from StyledHost in payments-list-core. 

Could you please explain this change when you have a moment? I'm curious

Reply:
This was unintended left behind. I added this while testing things, and forgot to remove. Thanks for catching that.